presto-main/src/main/java/io/prestosql/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -191,7 +198,7 @@ private Node rewriteShowStats(ShowStats node, Table table, List<SelectItem> sele
                 accessControl.checkCanSelectFromColumns(session.toSecurityContext(), tableMetadata.getQualifiedName(), extractNamedResultColumns);
             }
             catch (AccessDeniedException e) {
-                throw new AccessDeniedException(format("Cannot show stats for table %s", tableMetadata.getTable().toString()));
+                throw new AccessDeniedException(format("Cannot show stats for table %s", tableMetadata.getTable().getTableName()));
```

Comment:
If the table name was qualified in query, it should be qualified in error message

(there is no opposite implication IMO)